Index: src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>spring.application.name=ai\nspring.ai.openai.api-key=XXXX\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/src/main/resources/application.properties b/src/main/resources/application.properties
--- a/src/main/resources/application.properties	(revision ca475833444c187a949802a3256411a3894a7a46)
+++ b/src/main/resources/application.properties	(date 1712619573311)
@@ -1,3 +1,2 @@
 spring.application.name=ai
-spring.ai.openai.api-key=XXXX
 
